routers/web/repo/view.go

```diff
@@ -406,7 +406,7 @@ func renderFile(ctx *context.Context, entry *git.TreeEntry, treeLink, rawLink st
 	}
 	defer dataRc.Close()
 
-	ctx.Data["Title"] = ctx.Data["Title"].(string) + " - " + ctx.Repo.TreePath + " at " + ctx.Repo.BranchName
+	ctx.Data["Title"] = ctx.Data["Title"].(string) + " - " + ctx.Repo.TreePath + " at " + ctx.Repo.RefName
```

Comment:
ah I see that the locale change wasn't actually adding a new string but rather making it translatable. I think it would be better to put it back in :confounded:

Reply:
We could let it be. Because this maybe the last version of v1.15 and v1.16 will have the translation.